iset.mm

```diff
@@ -28761,6 +28761,11 @@ subset operation on the right hand side could be an equality (that is,
     eqv $p |- ( A = _V <-> A. x x e. A ) $=
       ( cvv wceq cv wcel wb wal dfcleq vex tbt albii bitr4i ) BCDAEZBFZNCFZGZAH
       OAHABCIOQAPOAJKLM $.
+
+    $( A class which is not inhabited is empty.  (Contributed by Jim Kingdon,
+       1-Jul-2022.) $)
+    nm0 $p |- ( -. E. x x e. A -> A = (/) ) $=
```

Comment:
maybe state it as a biconditional ?

Reply:
Seems like a good idea. I'll have to change the proof which depends on it (in #2659 ) but that won't be hard.

Reply:
I updated this pull request with making this biconditional (and renaming it to `notm0` to avoid a naming clash with set.mm).

I also copied http://us.metamath.org/mpeuni/nel0.html from set.mm because it is a similar theorem which iset.mm doesn't have yet but where the set.mm proof works unmodified.

I should probably stop now before I make this pull request even bigger than it is.